src/autogenerated/java/net/minestom/server/sound/BlockSoundTypes.java

```diff
@@ -0,0 +1,251 @@
+package net.minestom.server.sound;
+
+import java.util.HashMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * AUTOGENERATED by BlockSoundTypesGenerator
+ */
+public interface BlockSoundTypes {
+    HashMap<String, BlockSoundType> SOUND_TYPE_INDEX = new HashMap<>();
+
+    BlockSoundType EMPTY = new BlockSoundTypeImpl("empty", 1.0f, 1.0f, BuiltinSoundEvent.get("minecraft:intentionally_empty"), BuiltinSoundEvent.get("minecraft:intentionally_empty"), BuiltinSoundEvent.get("minecraft:intentionally_empty"), BuiltinSoundEvent.get("minecraft:intentionally_empty"), BuiltinSoundEvent.get("minecraft:intentionally_empty"));
```

Comment:
Are the values listed in a JSON file?

Reply:
Yes, they're part of the MinestomDataGenerator's output

Reply:
Like with generator.generateKeys?

That could work, but I'd need to edit how the JSON is generated on the DataGen side a bit, and again I don't think BlockSounds would work with a registry.

They still aren't network data that needs to be synced to a client.

Reply:
I adjusted the Data Geneator and would propose something like that:
```java
interface BlockSoundTypes {
    BlockSoundType EMPTY = BlockSoundTypeImpl.get("minestom:empty");

    BlockSoundType WOOD = BlockSoundTypeImpl.get("minestom:wood");

    BlockSoundType GRAVEL = BlockSoundTypeImpl.get("minestom:gravel");

    BlockSoundType GRASS = BlockSoundTypeImpl.get("minestom:grass");
    // ... 
}
```